components/user-store/org.wso2.carbon.identity.user.store.count/src/main/java/org/wso2/carbon/identity/user/store/count/UserStoreCountService.java

```diff
@@ -71,32 +74,31 @@
      * @return the number of roles matching the filter by each domain
      */
     public PairDTO[] countRoles(String filter) throws UserStoreCounterException {
+
         Set<String> userStoreDomains = UserStoreCountUtils.getCountEnabledUserStores();
-        //add 3 more for the counts of Internal, Application domains
+        //add 2 more for the counts of Internal, Application domains
```

Comment:
Make the first letter of the comment capital, and end the comment with a full stop.